tests/system/Router/RouteCollectionTest.php

```diff
@@ -970,10 +970,11 @@ public function testAddRedirect()
 	{
 		$routes = $this->getCollector();
 
-		$routes->addRedirect('users', 'Users::index', 307);
+		//The second parameter is either the new URI to redirect to, or the name of a named route.
+		$routes->addRedirect('users', 'users/index', 307);
```

Comment:
I think instead of modify existing test, it needs new tests for new allowed usage?

Reply:
This not a new allowed usage. The test is a mistak,It should not use Class::method.This is not what we accept